Dashboard/app/js/lib/views/reports/export-reports-views.js

```diff
@@ -66,7 +71,10 @@ FLOW.ReportLoader = Ember.Object.create({
       });
     }
 
-    criteria.opts.lastCollection = '' + (exportType === 'DATA_CLEANING' && FLOW.selectedControl.get('selectedSurveyGroup').get('monitoringGroup') && !!FLOW.editControl.lastCollection);
+    var exportOption = exportType === 'DATA_CLEANING' ? this.cleaningExportOption : exportType === 'DATA_ANALYSIS' ? this.analysisExportOption : "" ;
```

Comment:
The nested ternary statement looks a little strange. Since there are more than two options here perhaps we can instead use an `if ... else if ... else` statement to make it more readable.